package.json

```diff
@@ -66,12 +66,12 @@
     "@edx/browserslist-config": "^1.1.1",
     "@edx/frontend-build": "12.8.57",
     "@edx/reactifex": "2.2.0",
-    "@testing-library/react": "11.2.7",
+    "@testing-library/react": "12.1.5",
+    "@wojtekmaj/enzyme-adapter-react-17": "0.8.0",
```

Comment:
Given that this is now an [archived repo](https://github.com/wojtekmaj/enzyme-adapter-react-17), is the plan to remove this once we bump this repo to v18?

Reply:
Yes, this will be removed once this repo is bumped to v18.